lib/instances/BindableFunction.lua

```diff
@@ -0,0 +1,52 @@
+local BaseInstance = import("./BaseInstance")
+local InstanceProperty = import("../InstanceProperty")
+
+local BindableFunction = BaseInstance:extend("BindableFunction", {
+	creatable = true,
+})

```

Comment:
The default name of a `BindableFunction` is just `Function` for some reason, so we need to handle that in the intializer.

Reply:
Fixed